chef-utils/lib/chef-utils/dsl/platform_family.rb

```diff
@@ -86,6 +86,17 @@ def macos?(node = __getnode)
       # chef-sugar backcompat method
       alias_method :mac_os_x?, :macos?
 
+      # Determine if the Ruby VM is currently running on a Mac node (This is useful primarily for internal use
+      # by chef client before the node object exists).
```

Comment:
```suggestion
      # by Chef Infra Client before the node object exists).
```

This stuff ends up in the docs so the branding matters even if it's internal